daisy_workflows/build-publish/windows_install_media/windows-install-media.publish.json

```diff
@@ -0,0 +1,38 @@
+{{/*
+  Template to publish Windows install media images.
+  By default this template is setup to publish to the 'gce-image-builder'
+  project, the 'environment' variable can be used to publish to 'test', 'prod',
+  or 'staging'.
+  DeleteAfter is set to 180 days for all environments other than prod where no
+  time period is set.
+*/}}
+{
+  "Name": "windows-install-media",
+  {{$work_project := printf "%q" "gce-image-builder" -}}
+  {{$delete_after := `"24h*30*6"` -}}
+  {{$endpoint := `"https://www.googleapis.com/compute/alpha/projects/"` -}}
+  {{if eq .environment "test" -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "bct-prod-images",
+  "ComputeEndpoint": {{$endpoint}},
+  "DeleteAfter": {{$delete_after}},
+  {{- else if eq .environment "prod" -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "compute-image-tools",
+  "ComputeEndpoint": {{$endpoint}},
+  "DeleteAfter": {{$delete_after}},
+  {{- else -}}
```

Comment:
Can you share why we'd want this to publish to gce-image-builder?

If there isn't a specific need for it in that project, I'd suggest having the else condition use bct-prod-images and then the if "test" can be removed as well.

Reply:
No Special reason, It's just like that for long. Let me change.